grafana/datasource.yml

```diff
@@ -6,15 +6,14 @@ datasources:
   url: $DATABASE_HOST:$DATABASE_PORT
   user: $DATABASE_USER
   database: $DATABASE_NAME
-  password: ""
```

Comment:
Is it safe to obmit the password field?

Reply:
Yes. secureJsonData.password should be enough.